addons/binding/org.openhab.binding.heos/ESH-INF/thing/HeosGroup.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="group">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+		<label>HEOS Group</label>
+		<description>A group of HEOS Player</description>
+
+		<channels>
+			<channel id="Control" typeId="control" />
+			<channel id="Volume" typeId="volume" />
+			<channel id="Mute" typeId="mute" />
+			<channel id="Inputs" typeId="input" />
+			<channel id="Title" typeId="title" />
+			<channel id="Artist" typeId="artist" />
+			<channel id="Album" typeId="album" />
+			<channel id="Cover" typeId="cover" />
+			<channel id="CurrentPosition" typeId="currentPosition" />
+			<channel id="Duration" typeId="duration" />
+			<channel id="Type" typeId="type" />
+			<channel id="Station" typeId="station" />
+			<channel id="PlayUrl" typeId="playUrl" />
+			<channel id="Ungroup" typeId="ungroup" />
+			<channel id="Shuffel" typeId="shuffleMode" />
+			<channel id="RepeatMode" typeId="repeatMode" />
+			<channel id="Playlists" typeId="playlists" />
+		</channels>
+		<config-description>
```

Comment:
These seem to be far too many config parameters. I would assume that all that is required to identify the group (and make the handler work) is the gid. The other values could be added as properties, but I don't see the need for the user to specify any of these.